### PR TITLE
Fix uninitialized value $s warning in windows static builds

### PR DESCRIPTION
**Pull request by bernd-edlinger**
Fixes: #6826

[extended tests]

### CODE REVIEW

**Comment by mattcaswell** [comment]
Travis red cross appears unrelated.

**Comment by levitte** [comment]
-1

This isn't the right answer.  There are a number of `quotify1(shlib($whatever))`, and those will return a quoted empty string where they shouldn't.  The right answer in this case should be to change `quotify1(shlib` to `quotify_l(shlib` in `windows-makefile.tmpl`

**Comment by bernd-edlinger** [comment]
No problem, I can change the PR, or did you already have one?

**Comment by levitte** [comment]
Feel free to go ahead

**Comment by bernd-edlinger** [comment]
funny, `quotify_l(@{shlib($_)})` did not make a warning, but does not print anything when it should.
I think I need to do `my $l = shlib($_); quotify1($l) if defined($l);`

**Comment by bernd-edlinger** [comment]
okay, next try.

**Comment by bernd-edlinger** [comment]
CI is good now.

File: Configurations/windows-makefile.tmpl [hunk 1]
```diff
@@ -93,8 +93,8 @@ GENERATED={- # common0.tmpl provides @generated
                        @generated) -}
 
 INSTALL_LIBS={- join(" ", map { quotify1(lib($_)) } @{$unified_info{install}->{libraries}}) -}
-INSTALL_SHLIBS={- join(" ", map { quotify1(shlib($_)) } @{$unified_info{install}->{libraries}}) -}
-INSTALL_SHLIBPDBS={- join(" ", map { local $shlibext = ".pdb"; quotify1(shlib($_)) } @{$unified_info{install}->{libraries}}) -}
+INSTALL_SHLIBS={- join(" ", map { my $l = shlib($_); quotify1($l) if defined($l) } @{$unified_info{install}->{libraries}}) -}
```
Comment (levitte) [inline Configurations/windows-makefile.tmpl]: Why not just `quotify_l(shlib($_))`?

**Comment by levitte** [comment]
> funny, `quotify_l(@{shlib($_)})` did not make a warning, but does not print anything when it should.

Well, `@{shlib($_)}` expects `shlib` to return an array reference rather than an array...

File: Configurations/windows-makefile.tmpl (hunk 1 above)
Comment (bernd-edlinger) [reply Configurations/windows-makefile.tmpl]: me no speak perlish...

**Comment by bernd-edlinger** [comment]
It did unfortunately not cherry-pick, I will send a new PR for 1.1.0 in a moment.

**Pull request closed without merging**